packages/react-components/src/components/utils/GridLayoutUtils.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlockProps } from '../GridLayout';
+
+const TARGET_RATIO = 16 / 9;
+const TOLERANCE_RATIO = 8 / 9;
+
+const isRatioBetterThan = (A: number, B: number, targetRatio?: number): boolean => {
+  const _targetRatio = targetRatio ?? TARGET_RATIO;
+  return Math.abs(_targetRatio - A) < Math.abs(_targetRatio - B);

```

Comment:
Wouldn't this be the same as
`return B < A` since target ratio is the same for both of them.

lets assign a constant to `targetRatio = 100`
If `A = 20` and `B = 30`
the original function returns `false` and so does `B < A`.
When `B = 20` and `A = 30`
the original function returns `true` and so does `B < A`.

This would only fail if there is a chance of A or B being negative. But from the way they are being generated, that doesn't look to be the case.

Please correct me if I am wrong.

Reply:
If `target ratio = 100` and `A = 105` and `B = 90`
`B < A` is false but `|100 - A| < |100 - B|` (5 < 10) is true.
Basically I'm asking if A is closer (ie. have a smaller absolute difference) to the target ratio than B
But yes A and B can't be negative.